core/src/main/java/org/apache/iceberg/rest/RESTTableOperations.java

```diff
@@ -103,8 +107,11 @@ public TableMetadata current() {
   @Override
   public TableMetadata refresh() {
     Endpoint.check(endpoints, Endpoint.V1_LOAD_TABLE);
-    return updateCurrentMetadata(
-        client.get(path, LoadTableResponse.class, headers, ErrorHandlers.tableErrorHandler()));
+    LoadTableResponse loadTableResponse =
+            client.get(path, LoadTableResponse.class, headers, ErrorHandlers.tableErrorHandler());
+    updateCurrentMetadata(loadTableResponse);
+    updateIo(loadTableResponse);
```

Comment:
we tend to do IO instead of Io when we camel case. so 
```suggestion
    updateIO(loadTableResponse);
```